nucleus/admin/cli/pom.xml

```diff
@@ -120,6 +120,9 @@
                         <manifest>
                             <mainClass>com.sun.enterprise.admin.cli.AdminMain</mainClass>
                         </manifest>
+                        <manifestEntries>
```

Comment:
There are duplicate jars for bean-validator and bean-validator-cdi.
We are removing hibernate-validator and hibernate-validator-cdi jars and continue to bundle bean-validator and bean-validator-cdi (custom made jars that has all dependencies).
admin-cli module and its downstream dependencies make use of hibernate validator. Today (before this change), the maven plugin adds all dependencies of "hk2" (hibernate validator being one of them) in its classpath. Since we are removing hibernate validator, we are explicitly adding bean-validator in the classpath. (bean-validator now being hosted as part of GlassFish modules, it is not auto detected during class-path entry detection and hence we are adding it explicitly).